lib/gym/options.rb

```diff
@@ -21,6 +21,10 @@ def self.plain_options
                                        UI.user_error!("Workspace file not found at path '#{v}'") unless File.exist?(v)
                                        UI.user_error!("Workspace file invalid") unless File.directory?(v)
                                        UI.user_error!("Workspace file is not a workspace, must end with .xcworkspace") unless v.include?(".xcworkspace")
+                                     end,
+                                     conflicting_options: [:project],
+                                     conflict_block: proc do |value|
+                                       UI.user_error!("You can only pass either a 'workspace' or a '#{value.key}', not both")
```

Comment:
@KrauseFx Current implementation works not only for `:project` but for other potential conflicts as well!

Otherwise, In case if new conflicting options will be added to `gym`(or we will find new tuples of conflicting options) user error message should be changed in order to reflect the real cause of the conflict.